services/src/main/scala/cromwell/services/healthmonitor/impl/CommonMonitoredSubsystems.scala

```diff
@@ -0,0 +1,49 @@
+package cromwell.services.healthmonitor.impl
+
+import cromwell.core.docker.{DockerCliClient, DockerCliKey}
+import cromwell.services.EngineServicesStore
+import cromwell.services.healthmonitor.HealthMonitorServiceActor.{MonitoredSubsystem, OkStatus, SubsystemStatus}
+
+import scala.concurrent.{ExecutionContext, Future}
+
+trait CommonMonitoredSubsystems {
```

Comment:
I can't remember why I made this a `Trait` mixin instead of an `Object` except that it solved a problem I was having. If that bothers anyone let me know and I'll try to at least un-solve it and see what people think.